core/jobs.py

```diff
@@ -167,6 +175,8 @@ def enqueue(cls, job_id, queue_name, additional_job_params=None):
 
         cls._post_enqueue_hook(job_id)
 
+
```

Comment:
Please remove the extra newlines. There should be only one newline separating adjacent methods.

In general, make sure to check your PR's "Files Changed" tab immediately after pushing, in order to ensure that the diffs are exactly what you expect them to be.